pandas/core/series.py

```diff
@@ -4630,6 +4630,17 @@ def isin(self, values) -> "Series":
         4     True
         5    False
         Name: animal, dtype: bool
+        
+        .. versionchanged:: 1.2.0
```

Comment:
you don't neeed the version added.
Also reword this to.

`Strings and integers are distinct and are therefore not comparable`

Reply:
Done.
I wanted to ask why are some of the checks failing? Am I doing something wrong?

Reply:
click thru to see why these are failign: https://github.com/pandas-dev/pandas/pull/39064/checks?check_run_id=1703810526

for example